src/Functions/greatCircleDistance.cpp

```diff
@@ -430,10 +430,10 @@ Calculates the distance between two points on the Earth's surface using [the gre
     }
     {
         FunctionDocumentation::Description description = R"(
-Similar to `greatCircleDistance` but calculates the distance on WGS-84 ellipsoid instead of sphere. This is a more precise approximation of the Earth Geoid.
-The performance is the same as for `greatCircleDistance` (no performance drawback). It is recommended to use `geoDistance` to calculate the distances on Earth.
```

Comment:
Here, `greatCircleDistance`, is actually the function that we are documenting isn't it? Unless we mean to say: `Similar to the geoDistance function but ......`  and `It offers the same performance as for geoDistance`, or something similar, the current sentence doesn't seem to make much sense to me. Unless I'm missing something.